includes/LoginRedirect.php

```diff
@@ -12,75 +12,59 @@ class LoginRedirect {
 	/**
 	 * Handles the redirect to onboarding
 	 *
-	 * @param [type] $redirect The requested redirect URL.
-	 * @param [type] $redirect_to The requested redirect URL via param.
-	 * @param [type] $user The logged in user object.
+	 * @param string $original_redirect The requested redirect URL
 	 * @return string
 	 */
-	public static function handle_redirect( $redirect, $redirect_to, $user ) {
+	public static function handle_redirect( $original_redirect ) {
+		// Don't redirect if user is not an admin
+		if ( ! current_user_can( 'manage_options' ) ) {
+			return $original_redirect;
+		}
```

Comment:
Thanks for mentioning this, I'll tag a release and we'll be sure to test. If it's not working we'll start looking there, and if it is, we can clean up permissions elsewhere too.

Reply:
@arunshenoy99 Good call out. Sorry, I left this PR up yesterday because I wanted your team to take a look at it for this reason. Evan is going to delete the module release tag right now. We'll get this issue fixed first.

We do need this resolved for Jarvis (net new go live begins in ~2 hours) otherwise no Jarvis signups will see onboarding at all.

Reply:
Yes, thanks! We're working to get a quicker-than-usual release out in order to support new sites on Jarvis and this release isn't a scheduled update - that will still be next week. Finding some issues though, so this won't be going out (yet). I'll delete the tag and release now, and if we get one ready today will be sure to bump the module version too.

Reply:
Thanks, please do let us know if we can help in any way.